unitTests/ui/ResponsiveComponents/ResponsiveTabsTest.ts

```diff
@@ -191,6 +202,45 @@ export function ResponsiveTabsTest() {
           });
         });
       });
+
+      describe('when the tab section is hidden', () => {
```

Comment:
I would be more specific in the description, `when the tab section has zero width`,

Reply:
Ok will do